models/tokens/optimism/tokens_optimism_erc20.sql

```diff
@@ -11,25 +11,16 @@ END
 AS is_counted_in_tvl
 FROM (
 	SELECT
-	coalesce(c.contract_address, t.contract_address, b.contract_address) AS contract_address
+	c.contract_address
```

Comment:
Does this produce the same output? Are all contract_address unique?

Reply:
we should be careful here for sure.
@MSilb7 since you built these -- can you help here? maybe review the optimism pieces, rather than looking to build yourself?

Reply:
Simplifying the `contract_address` field here makes sense. I think the coalesce was left over from old logic.

Reply:
all tokens should be in the `c` table, so it works (and they're left joined)